ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -1,43 +1,86 @@
+from api_testing.apis.test_plan_api import TestPlanAPI
+from ui_testing.pages.testplan_page import TstPlan
+from api_testing.apis.general_utilities_api import GeneralUtilitiesAPI
+from ui_testing.pages.contacts_page import Contacts
+from random import randint
+import random
+
+
+class OrdersTestCases(BaseTest):
+    def setUp(self):
+        super().setUp()
+        self.order_page = Order()
+        self.orders_api = OrdersAPI()
+        self.testplan_page = TstPlan()
+        self.orders_page = Orders()
+        self.analyses_page = AllAnalysesPage()
+        self.article_api = ArticleAPI()
+        self.test_unit_api = TestUnitAPI()
+        self.contacts_api = ContactsAPI()
+        self.single_analysis_page = SingleAnalysisPage()
+        self.test_plan_api = TestPlanAPI()
+        self.contacts_api = ContactsAPI()
+        self.general_utilities_api = GeneralUtilitiesAPI()
+        self.contacts_page = Contacts()
+        self.set_authorization(auth=self.contacts_api.AUTHORIZATION_RESPONSE)
+        self.order_page.get_orders_page()
+        self.orders_api.set_configuration()
+
+    def test068_choose_test_plans_without_test_units(self):
+        """
+        Orders: Create: Orders Choose test plans without test units
+        LIMS-4350
+        """
+        self.info(' Create new order.')
+        self.order_page.click_create_order_button()
+        self.order_page.set_new_order()
+        self.order_page.set_contact(contact='')
+        self.order_page.sleep_small()
+        self.order_page.set_material_type(material_type='Raw Material')
+        self.order_page.sleep_small()
+        order_no = self.order_page.get_no()
+        self.info('create multiple testplan with same article of each suborder')
+        tst_plan1, tst_plan2, article1 = TestPlanAPI().create_multiple_test_plan_with_same_article()
+        tst_plan3, tst_plan4, article2 = TestPlanAPI().create_multiple_test_plan_with_same_article()
+        tst_plan5, tst_plan6, article3 = TestPlanAPI().create_multiple_test_plan_with_same_article()
+        self.order_page.set_article(article=article1)
+        self.order_page.sleep_small()
+        self.order_page.set_test_plan(test_plan=tst_plan1)
+        self.order_page.set_test_plan(test_plan=tst_plan2)
+        self.info('create 3 suborder')
+        self.order_page.create_new_suborder(material_type='Raw Material', test_plans=[tst_plan3, tst_plan4],
+                                            article_name=article2,add_tstunit=False)
+        self.order_page.create_new_suborder(material_type='Raw Material', test_plans=[tst_plan5, tst_plan6],
+                                            article_name=article3,add_tstunit=False)
+
+        self.order_page.save(save_btn='order:save_btn')
+        order_id = self.order_page.get_order_id()
+        suborders_data,_ = self.orders_api.get_suborder_by_order_id(order_id)
+        suborders = suborders_data['orders']
+        analysis_no = []
```

Comment:
assertion should be in GUI, 
1) filter by order no and assert that 3 analysis in child table
2) Navigate to analysis tab and filter by order no, you should get 3 different analysis
check that each analysis has 2 test units that relates to selected test plan